uhdm-plugin/UhdmAst.cc

```diff
@@ -841,30 +841,43 @@ static void simplify_format_string(AST::AstNode *current_node)
 
 static void simplify(AST::AstNode *current_node, AST::AstNode *parent_node)
 {
+    if (current_node->type == static_cast<int>(AST::AST_DOT))
+        current_node->type = AST::AST_IDENTIFIER;
```

Comment:
Why we need to do this? Is this because you later simplify ``dot`` to get constant?

If it is the case, please remove this and create temporary ``AST_IDENTIFIER`` and simplify it instead.

Reply:
Done